### DISCUSSION TITLE
PHP FORM HANDLNG

### DISCUSSION THREAD

**Discussion by etefiajnr** [filamentphp/filament#5259] [Help]
Please I need a php script/code that will save my below form input data to my log.txt (solo.txt) using the post method, I am really stucked so I need anybody's idea. 

<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title>LOGIN FORM</title>
<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script> 
<style>
	.login-form {
		width: 400px;
    	margin: 50px auto;
	
  }
    .login-form form {
    	margin-bottom: 15px;
        background: #ff00ff;
        box-shadow: 3px 3px 3px rgba(0, 0, 0, 0.3);
        padding: 20px;
      border-radius: 8px
              
    }
    .login-form h2 {
        margin: 0 0 15px;
    }
    .form-control, .btn {
        min-height: 45px;
        border-radius: 5px;

  }
    .btn {        
        font-size: 20px;
        font-weight: bold;
      border-radius: 8px;
      background: pink;
    }
</style>
</head>
<body style="background-color:#ffffff">
 
<div class="login-form">
    <form action="solo.php" method="post">
        <h4 class="text-center">Login here</h2>       
        <div class="form-group">
            <input type="text" class="form-control" placeholder="Email or phone" required="required"
                 >
        </div>
        <div class="form-group">
            <input type="password" class="form-control" placeholder="Password" required="required">
        </div>
        <div class="form-group">
            <button type="submit" class="btn btn-primary btn-block">Log in</button>
        </div>
        <div class="clearfix">
            <label class="pull-left checkbox-inline"><input type="checkbox"> Remember me</label>
        </div>        
    </form>
   
</div>
</body>
</html>

**Comment by ArtMin96** [comment]
```php
Storage::put('log.txt', 'contents is written inside log.txt');
```

or use php `file_put_contents()` function

**Reply by etefiajnr** [reply]
Thanks for trying to help me, Please can you write it the best way for me please, I dont know how to writ it

**Reply by ArtMin96** [reply]
I don't really know what you want to save in the file. Use this, just set the content to whatever you want.

**Reply by ArtMin96** [reply]
Also, are you sure you are using filament?

**Reply by ArtMin96** [reply]
I think you wrote in the wrong place. Filament does not use jQuery or bootstrap.

**Reply by danharrin** [reply]
Yeah this is totally unrelated to Filament
